doc/api/deprecations.md

```diff
@@ -2143,6 +2143,9 @@ The `crypto._toBuf()` function was not designed to be used by modules outside
 of Node.js core and was removed.
 
 ### DEP0115: `crypto.prng()`, `crypto.pseudoRandomBytes()`, `crypto.rng()`
+
+<!--lint disable nodejs-yaml-comments -->
```

Comment:
Should the dual `pr-url` be allowed in the future, or better to ignore the one-off?

Reply:
I think that's a one off, we can still change that later if it becomes more common.